app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 4)) == 17
```

Comment:
The test is checking the sum of the parts, but it does not check whether the parts are equal or not. It would be better to also check if the parts are equal to ensure that the function is working correctly.